### PR TITLE
Add osds_nomemtune group

### PR DESCRIPTION
**Pull request by mnasiadka**
Automatically adds _no_autotune_memory label for those hosts.

### CODE REVIEW

**Comment by mnasiadka** [comment]
label documented here: https://docs.ceph.com/en/latest/cephadm/host-management/#special-host-labels

File: roles/cephadm/templates/cluster.yml.j2 [hunk 1]
```diff
@@ -17,6 +17,9 @@ labels:
 {% if host in groups.get('rgws', []) %}
   - rgw
 {% endif %}
+{% if host in groups.get('osds_nomemtune', []) %}
+  - _no_autotune_memory
```
Comment (markgoddard) [inline roles/cephadm/templates/cluster.yml.j2:21]: Would it be more flexible to allow specifying a list of labels for any host?

Comment (mnasiadka) [reply roles/cephadm/templates/cluster.yml.j2:21]: Might be, but it's a) more work b) change of approach c) we already manage _admin in that way more or less - I don't think there will be many more of those labels in future (and _no_schedule is rather a temporary thing, so we don't need to manage it here).

Comment (mnasiadka) [reply roles/cephadm/templates/cluster.yml.j2:21]: So basically what you propose is to have a host/group variables that is "cephadm_labels" and apply them in addition to those that we already have for mons/mgrs/osds/rgws?

**Review by markgoddard — CHANGES_REQUESTED** [review]

**Review by markgoddard — APPROVED** [review]

**Pull request merged**